scylla-cql/src/frame/response/new_type_parser.rs

```diff
@@ -0,0 +1,308 @@
+use crate::frame::frame_errors::CustomTypeParseError;
+use std::{borrow::Cow, char, str::from_utf8};
+
+use super::result::NativeType;
+use super::result::{ColumnType, UserDefinedType};
+use std::sync::Arc;
+
+use super::result::CollectionType;
+use crate::utils::parse::ParserState;
+use either;
```

Comment:
Those imports are quite mixed up: you have multiple groups of imports, each has both crate imports and std/external imports.
We don't have an established import style in the project, mostly due to rustfmt not stabilizing import-related options for years. However, we should still try to do something reasonable.

Please either:
- Merge them into one group, and sort by std/crate/external. rust-analyzer should make that for you with the right settings
- Have a separate group for std / crate /external.

Apart from that, you have `use std::{borrow::Cow, char, str::from_utf8};`. Please avoid this, it is a pain when rebasing. Switch to `use` statement per module, or per item.